docs/source/configuration.rst

```diff
@@ -1887,25 +1887,26 @@ Based on the OGP metadata Converse will render a URL preview (also known as an
   the ``show_images_inline``, ``embed_audio`` and ``embed_videos`` settings.
 
 
-reuse_scram_keys
-----------------
+reuse_keys
+----------
```

Comment:
I'm not sure how best to handle this. I chose to repurpose reuse_scram_keys to also cover FAST because they are so similar. But there's other options

- add a separate `reuse_fast_keys` option
- make FAST always enabled, or maybe just a `fast` that defauls to `true`.